backend/api/models/FuelCode.py

```diff
@@ -0,0 +1,126 @@
+"""
+    REST API Documentation for the NRS TFRS Credit Trading Application
+
+    The Transportation Fuels Reporting System is being designed to streamline
+    compliance reporting for transportation fuel suppliers in accordance with
+    the Renewable & Low Carbon Fuel Requirements Regulation.
+
+    OpenAPI spec version: v1
+
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+"""
+from django.db import models
+from auditable.models import Auditable
+
+from api.models.FuelCodeStatus import FuelCodeStatus
+
+
+class FuelCode(Auditable):
+    """
+    Fuel codes for the Secure Document Upload
+    """
+    fuel_code = models.CharField(
+        max_length=100,
+        db_comment='Low Carbon Fuel Code.'
+    )
+    company = models.CharField(
```

Comment:
Would this be a free-form field or is there an opportunity to use the existing list of Organization names?

Reply:
While there might be a couple of companies that's the same as fuel supplier list, most of them would be very different. Since this is more of a list of fuel producers, rather than fuel suppliers